dags/hivemind_etl_helpers/tests/integration/test_load_envs.py

```diff
@@ -21,3 +22,18 @@ def test_mongo_envs_values(self):
         self.assertIsInstance(mongo_creds["password"], str)
         self.assertIsInstance(mongo_creds["host"], str)
         self.assertIsInstance(mongo_creds["port"], str)
+
+    def test_redis_envs_check_type(self):
+        redis_creds = load_redis_credentials()
+        self.assertIsInstance(redis_creds, dict)
+
+    def test_redis_envs_values(self):
+        redis_creds = load_redis_credentials()
+
+        self.assertNotEqual(redis_creds["password"], None)
+        self.assertNotEqual(redis_creds["host"], None)
+        self.assertNotEqual(redis_creds["port"], None)
+
+        self.assertIsInstance(redis_creds["password"], str)
+        self.assertIsInstance(redis_creds["host"], str)
+        self.assertIsInstance(redis_creds["port"], str)
```

Comment:
@polux0 Please have look at this comment. When we're loading the credentials, in case of not being available it is loading empty string, so please do another assertion to check if the values are not equal to empty string